premake5.lua

```diff
@@ -30,8 +30,19 @@ project "Hazel"
 	targetdir ("bin/" .. outputdir .. "/%{prj.name}")
 	objdir ("bin-int/" .. outputdir .. "/%{prj.name}")
 
-	pchheader "hzpch.h"
-	pchsource "Hazel/src/hzpch.cpp"
+	filter "action:vs2017"
```

Comment:
I think this can be changed to `filter "action:vs*"` to cover all versions of Visual Studio